### PR TITLE
add robot name and type args in app_manager

### PR DESCRIPTION
**Pull request by knorth55**
add `--robot-name` and `--robot-type` in `app_manager`

### CODE REVIEW

File: scripts/app_manager [hunk 1]
```diff
@@ -62,8 +64,16 @@ def main():
             plugin = yaml.safe_load(f)
         plugins += plugin
 
-    robot_name = rospy.get_param('/robot/name', 'robot')
-    robot_type = rospy.get_param("/robot/type", None)
+    if args.robot_name is None:
```
Comment (k-okada) [inline scripts/app_manager:67]: we're expecting each robot have `name` and `type` as param https://github.com/pr2-debs/pr2-core/blob/master/root/etc/ros/robot.yaml, https://github.com/fetchrobotics/fetch_robots/pull/22/files,

Comment (knorth55) [reply scripts/app_manager:67]: I want to launch multiple `app_manager`, such as robot and remote, in one ROS network.
In that case, it is necessary to set different `robot_name`, which is used for topic name and service in `app_manager` as here, so I open this PR.
https://github.com/PR2/app_manager/blob/kinetic-devel/src/app_manager/app_manager.py#L145-L154

Comment (knorth55) [reply scripts/app_manager:67]: However, there is also a problem to set the same robot name in `rwt_app_chooser`.
https://github.com/tork-a/visualization_rwt/blob/kinetic-devel/rwt_app_chooser/www/components/task.html#L198-L209

Comment (knorth55) [reply scripts/app_manager:67]: I understand the concept and design of `app_manager`, and I open several PRs.
`app_manager` is designed to run only 1 `app_manager` in 1 ROS network.
For remote usage, we need to app_manager other ROS network such as `localhost:11313`,
and connect to robot such as `localhost:11311` via `appmaster`.
I will close this PR, but I open #51 for the usage.

**Pull request closed without merging**